src/Extensions/Extensions.csproj

```diff
@@ -6,7 +6,8 @@
     <RootNamespace>Microsoft.Azure.Functions.Extensions</RootNamespace>
     <PackageId>Microsoft.Azure.Functions.Extensions</PackageId>
     <Description>Provides extensions to work with Azure Functions features.</Description>
-    <Version>1.2.0$(VersionSuffix)</Version>
+    <Version>1.2.0</Version>
```

Comment:
```suggestion
    <VersionPrefix>1.2.0</VersionPrefix>
```

Reply:
I had no idea, thank you for catching that